### PR TITLE
not persist version to settings.yaml resolve #3819

### LINKED ISSUE #3819: Not persist version, builddate and revisiondate to settings.yaml
These values should be baked in lantern binary.

Avoid persisting to avoid potential confusion. Say, when a new version of Lantern launched but didn't save `settings.yaml`, those values will be inconsistent with the actual version.

### PR DESCRIPTION
**Pull request by fffw**
Rather simple PR for #3819 

@myleshorton Do you think it's useful?

### CODE REVIEW

File: src/github.com/getlantern/detour/detour_test.go [hunk 1]
```diff
@@ -103,6 +103,7 @@ func TestRemoveFromWhitelist(t *testing.T) {
 	AddToWl(u.Host, false)
 	_, err := client.Get(mockURL)
 	if assert.Error(t, err, "should have error if reading times out through detour") {
+		time.Sleep(50 * time.Millisecond)
```
Comment (fffw) [inline src/github.com/getlantern/detour/detour_test.go]: Just to prevent Travis from complaining

**Comment by fffw** [comment]
Logged https://github.com/getlantern/lantern/issues/3822 for the failed test of Lantern embeddable client.

File: src/github.com/getlantern/detour/detour_test.go [hunk 2]
```diff
@@ -103,6 +103,7 @@ func TestRemoveFromWhitelist(t *testing.T) {
 	AddToWl(u.Host, false)
 	_, err := client.Get(mockURL)
 	if assert.Error(t, err, "should have error if reading times out through detour") {
```
Comment (shilingluo) [inline src/github.com/getlantern/detour/detour_test.go]: 1

**Comment by oxtoacart** [comment]
LGTM!

**Pull request merged**